src/popup/App.vue

```diff
@@ -136,6 +138,45 @@ body {
 @use '../styles/typography';
 @use '../styles/mixins';
 
+@keyframes scale-in {
+  0% {
+    opacity: 0;
+    transform: translateY(70%);
+  }
+  100% {
+    opacity: 1;
+    transform: none;
```

Comment:
transform: translateY(100%);

Reply:
I'm resetting the transform so `none` is totally valid here.